src/codemodder/codemodder.py

```diff
@@ -197,15 +172,5 @@ def run(original_args) -> int:
 def main():
     # TODO: I'm not sure why this needs to be parsed out separately
     # Maybe it has something to do with the invocation as python -m codemodder.
```

Comment:
I think you can entirely remove this comment. IIRC we parsed it first bc we wanted to send the args to another func, too. Now that's not the case